int8 terminate
---
string stop_status
